src/lib/abbot/nostr.py

```diff
@@ -1,25 +1,25 @@
-import os
+import IPython
 import uuid
-from pynostr.key import PrivateKey
+from pynostr.key import PrivateKey, PublicKey
 from pynostr.relay_manager import RelayManager
 from pynostr.filters import FiltersList, Filters
 from pynostr.event import EventKind
 from pynostr.encrypted_dm import EncryptedDirectMessage
 from pynostr.event import Event
 from typing import List, Optional
 
+from lib.abbot.env import BOT_NOSTR_PK
 from lib.abbot.exceptions.exception import try_except
-from lib.utils import try_get
 
-DM = EventKind.ENCRYPTED_DIRECT_MESSAGE  # 4
-CHANNEL_CREATE = EventKind.CHANNEL_CREATE  # 40
-CHANNEL_META = EventKind.CHANNEL_META  # 41
-CHANNEL_MESSAGE = EventKind.CHANNEL_MESSAGE  # 42
-CHANNEL_HIDE = EventKind.CHANNEL_HIDE  # 43
-CHANNEL_MUTE = EventKind.CHANNEL_MUTE  # 44
-BOT_CHANNEL_INVITE = 21021
+DM: EventKind = EventKind.ENCRYPTED_DIRECT_MESSAGE  # 4
+CHANNEL_CREATE: EventKind = EventKind.CHANNEL_CREATE  # 40
+CHANNEL_META: EventKind = EventKind.CHANNEL_META  # 41
+CHANNEL_MESSAGE: EventKind = EventKind.CHANNEL_MESSAGE  # 42
+CHANNEL_HIDE: EventKind = EventKind.CHANNEL_HIDE  # 43
+CHANNEL_MUTE: EventKind = EventKind.CHANNEL_MUTE  # 44
+BOT_CHANNEL_INVITE: EventKind = EventKind.BOT_CHANNEL_INVITE  # 21021
```

Comment:
i know, but since were making edits to the pynostr package anyway, i added it to that package. glad you called this out bc i need to update the requirements.txt to point to our forked version of pynostr